pages/common/flutter.md

```diff
@@ -0,0 +1,23 @@
+# flutter
+
+> Google’s free and open source cross-platform mobile app SDK.
+
+- Check the Flutter version:
+
+`flutter --version`
+
+- Call general help:
```

Comment:
Perhaps `Display general help` would read better?

Reply:
That's right, that sounds better.